### DISCUSSION TITLE
Is it possible to stop running task?

### DISCUSSION THREAD

**Discussion by iakov-kaiumov** [taskiq-python/taskiq#307] [Q&A]
Is it possible to stop running task by task id?

**Comment by devraj** [comment]
> Is it possible to stop running task by task id?

Could you elaborate your use case?

If you are within the execution block then nothing should stop you from returning if it matches your use case to stop execution?

Unless of course if you wish to signal the task from running from outside the task code block?

**Comment by iakov-kaiumov** [comment]
> > Is it possible to stop running task by task id?
> 
> Could you elaborate your use case?
> 
> If you are within the execution block then nothing should stop you from returning if it matches your use case to stop execution?
> 
> Unless of course if you wish to signal the task from running from outside the task code block?

In my use case I have a long-running task and I want to stop it from outside the task code block. I use Redis backend.
